README.md

```diff
@@ -109,6 +109,12 @@ image = 'custom-image'
 variants = ['gpu']
 ```
 
+One useful docker flag is the `-v` flag, that can be used at runtime to mount other directories not included in your image.  The syntax for the argument of `-v` is `source:destination`.  For example, if you have a large dataset in your home directory that you don't want to include as part of your image, you can mount it at runtime by addding the following line to your `config.toml` file:
+
+```toml
+docker_flags = [ '-v', '/home/_your_username_/dataset:/home/_your_username_/dataset' ]
```

Comment:
I wonder if it would look cleaner instead of doing `_your_username_` to instead just do `$USER`. This would also make the command copy-pastable.

Reply:
Can $HOME be used?

Reply:
That's probably an even better idea. Good call @crawfxrd

Reply:
@brs17 @crawxfrd 
Actually that's what I tried first, but neither $HOME nor $USER are allowed.  Try it and see:
```
docker: Error response from daemon: invalid volume specification: '$USER/datasets:$USER/dataset': invalid mount config for type "volume": invalid mount path: '$USER/dataset' mount path must be absolute
```

```
docker: Error response from daemon: invalid volume specification: '$HOME/datasets:$HOME/datasets': invalid mount config for type "volume": invalid mount path: '$HOME/datasets' mount path must be absolute.
```

Reply:
And if you try using $USER or $HOME on the source but not the destination as in
`docker_flags = [ '-v', '$USER/dataset:/dataset']
`, then you get an error like:

```
docker: Error response from daemon: create $USER/dataset: "$USER/dataset" includes invalid characters for a local volume name, only "[a-zA-Z0-9][a-zA-Z0-9_.-]" are allowed. If you intended to pass a host directory, use absolute path.

```

"use absolute path".

Reply:
In that case, `_your` could be dropped.

Beside that, I typically see angle brackets used for required arguments: `/home/<username>/<dataset>`